### ISSUE TITLE
Admin surveillance supervisor cant delete samples data

### ISSUE THREAD

**Issue by Iheanacho2027** [SORMAS-Foundation/SORMAS-Project#3881]
### Bug Description
Samples and Events data cant be deleted using the admin surveillance supervisor like it is currently with cases. This shouldnt be so @bernardsilenou  
### Steps to Reproduce
1.
2.

### System Details
* Device:
* SORMAS version: 1.53
* Android version/Browser:
* Server URL:
* User Role:

**Comment by bernardsilenou** [comment]
- Admin surveillance supervisors should be able to delete samples linked to entities that they have full access to. They should also be able to delete entities that they have full access to.
- @Iheanacho2027  Please can you add more information to this ticket, version, server, user role. Please test with version 1.53

**Comment by Iheanacho2027** [comment]
@bernardsilenou it occurs in 1.53.0. The "delete" button as displayed by the arrow button in the image below is missing in the page for a created sample linked to a case, on the Admin Surveillance supervisor user account.
![Uploading image.png…]()

**Comment by MateStrysewske** [comment]
This is not a bug since it was never part of the specification of the admin surveillance supervisor role (#2316).

**Comment by Iheanacho2027** [comment]
@MateStrysewske Yeah, but the admin surveillance supervisor should be able to do all an admin should do with exception to user creation and restriction to state of operation

**Comment by MateStrysewske** [comment]
Noted, I've simply changed the issue to a change request ;)
